Comment from PID-10/CWE-4 has been copied to PID-10/CWE-6 and PID-10/… (#159)
IHE/DEV.SDPi: asciidoc/volume2/gateways/tf2-ch-b-gateway-pid-mapping.adoc

|Alternate Identifier
|pm:PatientContextState+++<wbr/>+++/pm:CoreData+++<wbr/>+++/pm:Race+++<wbr/>+++/pm:Translation
/@Code

> |Note that if *pm:Race/@Code* contains a private code, the corresponding translation is to be mapped. In this case, only the first entry of the *pm:Translation* element list is to be mapped.

Reviewer: Otherwise was correct. The first translation shall only be mapped if it is NOT a private code.

> |Note that if *pm:Race/@Code* contains a private code, the corresponding translation is to be mapped. In this case, only the first entry of the *pm:Translation* element list is to be mapped.

Reviewer: ```suggestion
|Note that if *pm:Race/@Code* contains a private code, the corresponding translation is to be mapped. Otherwise only the first entry of the *pm:Translation* element list is to be mapped.

```

|PID-10/CWE-6
|Name of Alternate Coding System
|pm:PatientContextState+++<wbr/>+++/pm:CoreData+++<wbr/>+++/pm:Race+++<wbr/>+++/pm:Translation
/@CodingSystem

> |Note that if *pm:Race/@Code* contains a private code, the corresponding translation is to be mapped. In this case, only the first entry of the *pm:Translation* element list is to be mapped.

Reviewer: ```suggestion
|Note that if *pm:Race/@Code* contains a private code, the corresponding translation is to be mapped. Otherwise only the first entry of the *pm:Translation* element list is to be mapped.

```

|Alternate Coding System Version ID
|pm:PatientContextState+++<wbr/>+++/pm:CoreData+++<wbr/>+++/pm:Race+++<wbr/>+++/pm:Translation
/@CodingSystemVersion

> |Note that if *pm:Race/@Code* contains a private code, the corresponding translation is to be mapped. In this case, only the first entry of the *pm:Translation* element list is to be mapped.

Reviewer: ```suggestion
|Note that if *pm:Race/@Code* contains a private code, the corresponding translation is to be mapped. Otherwise only the first entry of the *pm:Translation* element list is to be mapped.

```
